### PR TITLE
6 skeleton site

### LINKED ISSUE #6: Create skeleton site.
As developers, we need an example skeleton site so we can get feedback and iterate on the design.

First design choice is to choose [Materials for Mkdocs](https://squidfunk.github.io/mkdocs-material/).

### PR DESCRIPTION
**Pull request by ccarouge**
Closes #6 

First iteration on the site design

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: publish
+on:
+  push:
+    branches: 
+      - 2-skeleton-site
```
Comment (aidanheerdegen) [inline .github/workflows/ci.yml:5]: Will this be changed to main after merging?

Comment (ccarouge) [reply .github/workflows/ci.yml]: This needs to change to main before merging but I don't know how to show off the site without this in as is. Which is annoying...

File: mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+site_name: ACCESSHub Demo
```
Comment (aidanheerdegen) [inline mkdocs.yml:1]: I guess we should take out Demo?

Comment (ccarouge) [reply mkdocs.yml:1]: Actually, I would leave it until we are ready to release it to the community. For me, it acts as a warning it is a work in progress. I should actually add a WIP warning in the home.

**Pull request merged**